src/content/doc-surrealdb/index.mdx

```diff
@@ -35,8 +47,105 @@ import DarkGettingStarted from "@img/image/dark/gettingstarted.png";
+ <div class="space-y-4">
+                <h2 class="text-4xl mt-10 font-thin">
+                    Explore Documentation
+                </h2>
+                <Boxes wider class="pt-2">
+                    <IconBox
+                        title="Querying"
+                        description="In this section, you will explore some of the ways you can query data in SurrealDB using SurrealQL, GraphQL or via any of the available SDKs. Regardless of the method you choose, SurrealDB provides powerful querying capabilities that allow you to retrieve, filter, and manipulate data efficiently and effectively."
+                        href="/docs/surrealdb/querying"
+                        icon={{ light: LightSidekick, dark: DarkSidekick }}
+                    />
+                    <IconBox
+                        title="Security"
+                        description="SurrealDB provides a robust security model that allows you to control access to your database. You can define roles and permissions for users and groups, and you can also use SurrealDB's authentication system to authenticate users and groups."
+                        href="/docs/surrealdb/security"
+                        icon={{ light: LightSidekick, dark: DarkSidekick }}
+                    />
+                    <IconBox
+                        title="Deployment"
+                        description="Learn how to deploy SurrealDB in minutes, and how to scale your database to meet the needs of your application."
+                        href="/docs/surrealdb/deployment"
+                        icon={{ light: LightSidekick, dark: DarkSidekick }}
+                    />
+                    <IconBox
+                        title="Embedding"
+                        description="SurrealDB is designed to be run in many different ways, and environments. Due to the separation of the storage and compute layers, SurrealDB can be run in embedded mode, from within your environments."
+                        href="/docs/surrealdb/embedding"
+                        icon={{ light: LightSidekick, dark: DarkSidekick }}
+                    />
+                    <IconBox
+                        title="CLI"
+                        description="Learn about SurrealDB's command line interface (CLI) that allows you to interact with your database. You can use the CLI to create and manage databases, tables, and records, as well as to execute queries and perform other database operations."
+                        href="/docs/surrealdb/cli"
+                        icon={{ light: LightSidekick, dark: DarkSidekick }}
+                    />
+                    <IconBox
+                        title="Reference guides"
+                        description="SurrealDB provides a variety of reference guides that cover a wide range of topics, from the basics of SurrealDB to more advanced concepts."
```

Comment:
```suggestion
                        description="Explore a variety of reference guides that cover a wide range of topics, from the basics of SurrealDB to more advanced concepts."
```

Suggesting this to avoid the duplication of "SurrealDB" in a single sentence